subsys/net/lib/openthread/platform/radio.c

```diff
@@ -1223,3 +1252,23 @@ otError otPlatRadioConfigureEnhAckProbing(otInstance *aInstance, otLinkMetrics a
 }
 
 #endif /* CONFIG_OPENTHREAD_LINK_METRICS_SUBJECT */
+
+otError otPlatRadioSetChannelMaxTransmitPower(otInstance *aInstance, uint8_t aChannel,
+					      int8_t aMaxPower)
+{
+	ARG_UNUSED(aInstance);
+	otError error = OT_ERROR_NONE;
+
+	if (aChannel >= OT_RADIO_2P4GHZ_OQPSK_CHANNEL_MIN &&
+		aChannel <= OT_RADIO_2P4GHZ_OQPSK_CHANNEL_MAX) {
```

Comment:
Would be nice to align to the opening bracket, currently it blends with the code.

Reply:
fixed